docgen/src/guides/advanced-topics.md

```diff
@@ -153,3 +153,127 @@ class App extends Component {
 export default withRouter(App);
 ```
 
+## React Native
```

Comment:
Yes I agree but then the How-To topics are smaller so it might be a little too much for those no?